sdk/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export * as config from './config';
 export * as blockchainData from './blockchain_data';
-export * as immutablexClient from './immutablex_client';
+export * as xClient from './x_client';
 export * as passport from './passport';
 export * as orderbook from './orderbook';
-export * as provider from './provider';
```

Comment:
@CodeSchwert this will quite likely affect TokenTrove. Has this been brought up to the Passport/Traders/Checkout PMs?

Reply:
I've given Steven Bloom (@sbloom-immu) a heads up.

Reply:
hey @naveen-imtb let me also flag this with Harris

Reply:
checked that TT did not use imx stuff in Prod.
we found a few other game/marketplace company used this. 
decision still push ahead because once we are v1, it'd be way much huge communication and usage fly in, will become impossible to change.